edx_django_utils/monitoring/internal/backends.py

```diff
@@ -145,6 +161,10 @@ def record_exception(self):
     def create_span(self, name):
         return self.dd_tracer.trace(name)
 
+    def tag_root_span_with_error(self, exception):
+        root_span = self.dd_tracer.current_root_span()
+        root_span.set_exc_info(type(exception), exception, exception.__traceback__)
```

Comment:
Is it possible to mock this instead to get a deeper mock and to fix this codecov comment?